app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,51 @@
+class NotesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_note, only: [:show, :edit, :destroy, :update]
+
+  def index
+    @notes = current_user.notes
+  end
+
+  def new
+    @note = current_user.notes.new
+  end
+
+  def create
+    @note = current_user.notes.create(note_params)
+    if @note.save
+      redirect_to [current_user, @note], notice: "Note has successfully created."
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show; end
+
+  def edit; end
+
+  def update
+    if @note.update(note_params)
+      redirect_to @note
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    if @note.destroy
+      redirect_to root_path
+    else
+      redirect_to @note
```

Comment:
should be one liner along-with `redirect` statement as `, alert: @note.errors.full_messages`

Reply:
done